include/mbedtls/entropy_poll.h

```diff
@@ -55,18 +55,47 @@ extern "C" {
 #endif
 
 #if !defined(MBEDTLS_NO_PLATFORM_ENTROPY)
+/**
+ * \brief           Platform-specific entropy initialization
+ *
+ * Call this function to initialize the platform entropy sources. On Unix-like
+ * platform, this function opens `/dev/urandom`.
+ *
+ * Calling this function directly is optional: mbedtls_platform_entropy_poll()
+ * calls mbedtls_platform_entropy_init() if needed.
+ */
+int mbedtls_platform_entropy_init( void );
+
 /**
  * \brief           Platform-specific entropy poll callback
+ *
+ * This function is meant to be registered with mbedtls_entropy_add_source().
```

Comment:
I think it needs to be clearer that this is platform code, and can be implemented by the porter if necessary.

Reply:
I've pushed a documentation update. I'm awaiting your response on the other requested changes.

Reply:
I'm fine with the documentation changes you made